src/bin/agent.rs

```diff
@@ -31,18 +35,38 @@ struct Arguments {
 #[tokio::main]
 async fn main() -> Result<()> {
     // Initialize a Tracing Subscriber
-    let fmt_builder = tracing_subscriber::fmt()
+    let fmt_layer = tracing_subscriber::fmt::layer()
         .with_file(false)
         .with_line_number(true)
         .with_thread_ids(true)
-        .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
+        .with_target(true)
```

Comment:
why is this :?

Reply:
ah good catch, seems like default is true, so we can remove this line https://github.com/tokio-rs/tracing/blob/master/tracing-subscriber/src/fmt/format/mod.rs#L600